### DISCUSSION TITLE
Adding sbml file to pickled community

### DISCUSSION THREAD

**Discussion by Montazar1234** [micom-dev/micom#140] [Q&A]
Hi,

I'm not sure if it's possible for me to add another model (via an SBML file) to an already existing community I generate with the build function? Or if I would need to generate the community object with all the models together at once. Any help or advice would be appreciated! 

Thanks,
Monty

**Comment by cdiener** [comment]
Right now, the easiest way is to rebuild from an amended taxonomy table. It's not impossible to add a model to an existing community, but it would be very cumbersome. We are working on changes that make it possible to additional models, specifically host tissues, to MICOM community models.
